sycl/test-e2e/ESIMD/unified_memory_api/slm_scatter_legacy.cpp

```diff
@@ -0,0 +1,20 @@
+//==------- slm_scatter_legacy.cpp - DPC++ ESIMD on-device test -----------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// Use per-kernel compilation to have more information about failing cases.
+// RUN: %{build} -fsycl-device-code-split=per_kernel -o %t.out
+// RUN: %{run} %t.out
+
+// The test verifies esimd::scatter() functions accepting  optional compile-time
+// esimd::properties. The scatter() calls in this test do not use VS > 1 (number
+// of loads per offset) to not impose using PVC features.
```

Comment:
please update here and in slm_scatter_legacy.cpp
```suggestion
// of stores per offset) to not impose using PVC features.
```